src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -741,6 +740,22 @@ public class SimpleEvents {
 				)
 				.since("2.7");
 
+		Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "[player] (wrong|unknown) (cmd|command) [send|use]")
+			.description("Called when the player uses a command that does not exist.")
+			.examples(
+				"on unknown command:",
+				"\tset event-unknown command message to \"Hey, this command does not exist.\""
+			)
+			.since("INSERT VERSION");
```

Comment:
your branch isn't clean, this is from your old pr.
You may want to restart this pr with a clean branch.